lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -689,6 +747,20 @@ module Resources
                       'start': '2016-12-08T21:53:53.000Z'
                     },
                     'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/period'
+                  },
+                  {
+                    'valueCodeableConcept': {
+                      'coding': [
+                        {
+                          'system': 'https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/40',
+                          'cod': '156',
```

Comment:
Missing an e on the end of 'code'.

Is there a way to add this standard coding in the relation extension as well?
{
'system': 'http://terminology.hl7.org/CodeSystem/v3-RoleCode',
'code': 'MTH',
'display': 'mother',
'userSelected': false
}

Reply:
added here https://github.com/cerner/fhir.cerner.com/pull/644/commits/767fd43d02089bf605dfdde72918fc393bec5655#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR1202-R1205

Reply:
+1 to standard codes added to all of the examples but write and patch.